_episodes/06-edges.md

```diff
@@ -75,23 +79,27 @@ def test_fib6():
     obs = fib(6)
     assert obs == 13)
 ~~~
+{: .python}
 
 Different functions will have different edge cases.
 Often, you need not test for cases that are outside the valid range, unless you
 want to test that the function fails.  In the `fib()` function negative and
-noninteger values are not valid inputs. Tests for these classes of numbers serve you well if you want to make sure that the function fails as
-expected. Indeed, we learned in the assertions section that this is actually quite a good idea.
+noninteger values are not valid inputs. Tests for these classes of numbers serve
+you well if you want to make sure that the function fails as expected. Indeed, we
+learned in the assertions section that this is actually quite a good idea.
 
-> ## Test for Graceful Failure {.challenge}
+> ## Implement these points
```

Comment:
Why change the title here? I don't think "Implement these points" is descriptive at all.